### PR TITLE
Separate into files, add acknowledgements page

### PR DESCRIPTION
**Pull request by bmagyar**
I tried to refrain from making changes to the content on this PR however there were some small typos fixes I did and I merged the previous "repositories" section with the top description of "packages" which was really referring to the repos, not the packages. It's more streamlined now.

I've added the acknowledgements page starting with PAL, we could discuss who else to add on the WG or slack.

### CODE REVIEW

File: acknowledgements.rst
Comment (destogl) [inline acknowledgements.rst:1]: You are using the same reference in all files. This is probably copy-paste issue/typo

File: acknowledgements.rst
Comment (destogl) [inline acknowledgements.rst:3]: What do you think to add this on "About" page something similar to [Navigation2](https://navigation.ros.org/about/index.html)

Why not also add list of the core maintainers? This would increase credibility, as there are persons with names behind the framework.

And we can link to GH statistics for the developers, saying "thanks" to all individuals working with us

**Review by destogl — APPROVED** [review]
Please change the references to pages. I trust you not to change the content :D 

Approving not to block this PR. Acknowledgment/About page can be then discussed after this refractoring

File: acknowledgements.rst
Comment (bmagyar) [reply acknowledgements.rst:3]: I quite like this idea although I prefer Moveit's format: https://moveit.ros.org/about/

Comment (bmagyar) [reply acknowledgements.rst:3]: opened #8

File: acknowledgements.rst
Comment (bmagyar) [reply acknowledgements.rst]: That fixed it, thanks!

**Pull request merged**
